commercetools-internal-docs/src/main/java/io/sphere/sdk/meta/ReleaseNotes.java

```diff
@@ -154,6 +154,10 @@
  </ul>
  -->
  -->
+ <h3 class=released-version id="v2_8_0">2.8.0 (04.04.2022)</h3>
+ <ul>
+ <li class=new-in-release>Support new Order message: {@link io.sphere.sdk.orders.messages.OrderPaymentAddedMessage}</li>
```

Comment:
please add the other changes too

Reply:
you're right I just find it out ;) I will update with the next commit.
Thanks